src/Equinox.MessageDb/MessageDb.fs

```diff
@@ -0,0 +1,401 @@
+﻿namespace Equinox.MessageDb
+
+open System
+open System.Text.Json
+open System.Threading.Tasks
+open Equinox.Core
+open FSharp.Control
+open FsCodec
+open Npgsql
+open Serilog
+
+type EventBody = JsonElement
+type EventData = IEventData<JsonElement>
+type IEventStoreConnection = MessageDbClient
+type ResolvedEvent = ITimelineEvent<JsonElement>
```

Comment:
again not sure this is a good idea to put in the impl here - some redundancy in the test suite is OK if it helps there